### PR TITLE
Updating ISSUE_TEMPLATES and PR TEMPLATE

### PR DESCRIPTION
**Pull request by rfelber**
This PR updates the project specific ISSUE_TEMPLATES and PR TEMPLATE to improve the community process.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new_security_scanner.md [hunk 1]
```diff
@@ -1,35 +1,41 @@
...
+<!--
+Hint: A general guide how to implement a new scanner is documented [here](https://github.com/secureCodeBox/secureCodeBox-v2/tree/master/docs/developer-guide)
+-->
+
+- [ ] Create a new folder with the name of the [scanner here](https://github.com/secureCodeBox/secureCodeBox-v2/tree/master/scanners), based on the existing template
+- [ ] Add a README.md and give a brief overview of the scanner and its configuration options.
+- [ ] Implement a new scanner specific scan-type.yaml
+- [ ] Implement a new scanner specific parse-definition.yaml
+- [ ] Add (optional) some cascading-rules.yaml
+- [ ] Add (optional) a Dockerfile for the scanner if there is now existing one public available on dockerHub
```
Comment (nigthknight) [inline .github/ISSUE_TEMPLATE/new_security_scanner.md]: I think there are two typos:
Add (optional) a Dockerfile for the scanner if there is no existing one publicly available on dockerHub

File: .github/ISSUE_TEMPLATE/new_hook.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+<!--
+Hint: A general guide how to implement a new scanner is documented [here](https://github.com/secureCodeBox/secureCodeBox-v2/tree/master/docs/developer-guide)
+-->
+
+- [ ] Create a new folder with the name of the [hook here](https://github.com/secureCodeBox/secureCodeBox-v2/tree/master/scanners), based on the existing template
+- [ ] Add a README.md and give a brief overview of the scanner and its configuration options.
+- [ ] Implement a new scanner specific scan-type.yaml
+- [ ] Implement a new scanner specific parse-definition.yaml
+- [ ] Add (optional) some cascading-rules.yaml
+- [ ] Add (optional) a Dockerfile for the scanner if there is now existing one public available on dockerHub
```
Comment (nigthknight) [inline .github/ISSUE_TEMPLATE/new_hook.md]: I think there are some typos:
Add (optional) a Dockerfile for the hook if there is no existing one publicly available on dockerHub

File: .github/ISSUE_TEMPLATE/new_hook.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+- [ ] Use the parserSDK to implement a new findings parser (currently based on NodeJS)
```
Comment (nigthknight) [inline .github/ISSUE_TEMPLATE/new_hook.md]: I'm not sure if this step is needed for a hook

File: .github/ISSUE_TEMPLATE/new_hook.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+- [ ] Add unit tests with at minimum 80% test coverage
+- [ ] Add some example scan.yaml and finding.yaml files in the example folder
```
Comment (nigthknight) [inline .github/ISSUE_TEMPLATE/new_hook.md:37]: I'm not sure if this step is needed for a hook

**Review by nigthknight — CHANGES_REQUESTED** [review]

**Review by J12934 — APPROVED** [review]
